clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/cache/Keys.groovy

```diff
@@ -194,8 +195,9 @@ class Keys {
   }
 
   static String getHealthCheckKey(String account,
+                                  String loadBalancerEndpoint,
                                   String healthCheckName) {
-    "$GoogleCloudProvider.GCE:${Namespace.HEALTH_CHECKS}:${account}:${healthCheckName}"
+    "$GoogleCloudProvider.GCE:${Namespace.HEALTH_CHECKS}:${account}:${loadBalancerEndpoint}:${healthCheckName}"
```

Comment:
As discussed offline, let's just drop the `compute#` prefix altogether and unify `kind` and `endpoint`.